build.gradle

```diff
@@ -6,7 +6,7 @@ plugins {
     id 'com.github.ben-manes.versions' version '0.12.0'
     id 'com.github.johnrengelman.shadow' version '1.2.3'
     id 'com.install4j.gradle' version '7.0.1'
-    id "de.qaware.seu.as.code.git" version "2.2.0"
+    id 'de.undercouch.download' version '3.2.0'
```

Comment:
3.3.0 is the latest version of this task, but there is a [known issue](https://github.com/michel-kraemer/gradle-download-task/issues/95) preventing its installation.